src/quo/components/list_items/network_list/view.cljs

```diff
@@ -0,0 +1,57 @@
+(ns quo.components.list-items.network-list.view
+  (:require
+    [quo.components.list-items.network-list.style :as style]
+    [quo.components.markdown.text :as text]
+    [quo.foundations.resources :as quo.resources]
+    [quo.theme :as quo.theme]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]))
+
+(defn- info
+  [{:keys [network-name label]}]
+  [rn/view {:style style/info}
+   [rn/image
+    {:source (quo.resources/get-network network-name)
+     :style  style/network-image}]
+   [rn/view
+    [text/text
+     {:weight          :semi-bold
+      :number-of-lines 1}
+     (if-not (empty? label) label "-")]]])
+
+(defn- values
+  [{:keys [token-value fiat-value theme]}]
+  [rn/view {:style style/values-container}
+   [text/text
+    {:weight          :medium
+     :size            :paragraph-2
+     :number-of-lines 1}
+    token-value]
+   [text/text
+    {:style           (style/fiat-value theme)
+     :size            :paragraph-2
+     :number-of-lines 1}
+    fiat-value]])
+
+(defn- view-internal
```

Comment:
I would suggest adding [malli](https://github.com/metosin/malli) schema for this component, You can refer this,

https://github.com/status-im/status-mobile/blob/31dea1ca13f0d2a9a29f16a69e459d6594af6c50/src/quo/components/wallet/required_tokens/view.cljs#L9-L23

https://github.com/status-im/status-mobile/blob/31dea1ca13f0d2a9a29f16a69e459d6594af6c50/src/quo/components/wallet/required_tokens/view.cljs#L54-L56

Reply:
oh nice @ajayesivan wasn't aware this exists, thanks for letting me know 👍  Have updated the code

Reply:
cc @clauxx maybe you can check the schema too ?